src/mmrelay/cli_utils.py

```
@@ -589,6 +589,53 @@ async def logout_matrix_bot(password: str):
+                credentials["user_id"] = user_id
+                from mmrelay.config import save_credentials
+
+                save_credentials(credentials)
+                logger.info("Updated credentials.json with fetched user_id")
+                print("✅ Updated credentials.json with fetched user_id")
+            else:
+                logger.error("Failed to fetch user_id from whoami response")
+                print("❌ Failed to fetch user_id from whoami response")
+
+        except asyncio.TimeoutError:
+            logger.error("Timeout while fetching user_id")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)